<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CourierWidgetClass</class>
 <widget class="QWidget" name="CourierWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CourierWidget</string>
  </property>
  <widget class="QListWidget" name="undeliveredListWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="fixedEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deliverButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mapButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
